Smoother scrolling with VM browser (RESUBMISSION)

DaveVoyles opened the pull request:
Original thread: https://github.com/MicrosoftDocs/azure-docs/pull/3152#issuecomment-333013220

Re-submitting after a large number of conflicts appeared in the previous one.

azurecla commented:
Hi __@gabrielweyer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
Thank you!

DaveVoyles committed:
Smoother scrolling with VM browser (RESUBMISSION)

Original thread: https://github.com/MicrosoftDocs/azure-docs/pull/3152#issuecomment-333013220

Re-submitting after a large number of conflicts appeared in the previous one.
